Formula/paket.rb

```diff
@@ -1,8 +1,8 @@
 class Paket < Formula
   desc "Dependency manager for .NET with support for NuGet and Git repositories"
   homepage "https://fsprojects.github.io/Paket/"
-  url "https://github.com/fsprojects/Paket/releases/download/5.249.0/paket.exe"
-  sha256 "b91f5125c8838de43137056ceb97f962218c47648b69ca3383c426a1d9ea01c2"
+  url "https://github.com/fsprojects/Paket/releases/download/5.249.1/paket.exe"
```

Comment:
a new release for this, https://github.com/fsprojects/Paket/releases/tag/5.249.2